client/app/lib/util/cleanup.coffee

```diff
@@ -38,3 +77,9 @@ module.exports = ->
     for option in options
       if kookies.expire key, option
         console.log "#{key} deleted"
+
+  for exp in regexWith
```

Comment:
What is wrong with `_mixpanel$` ? seems like works [here](http://coffeescript.org/#try:alert%20%2F%2F%2F_mixpanel%24%2F%2F%2F.test%20%22mp_113c2731b47a5151f4be44ddd5af0e7a_mixpanel%22)